client/coq.configuration.json

```
@@ -7,7 +7,6 @@
 	"brackets": [
 		["{", "}"],
 		["[", "]"],
-		["(*", "*)"],
 		["(", ")"]
 	]
-}
\ No newline at end of file
+}
```

```suggestion
,
	"autoClosingPairs": [
		{ "open": "{", "close": "}" },
		{ "open": "[", "close": "]" },
		{ "open": "(", "close": ")" },
		{ "open": "(*", "close": "*" }
	  ]
}
```

I think the change is sound, thanks! But I believe we can still keep the auto closing behavior for comments by adding this.

---

Seems good! I wonder if matching works the same here (if it will match both `"("` and `"(*"`). I'm not sure how to build and test this. If you tell me how to or validate it yourself, that'd be great!

Thanks for the quick response!

---

I tried it and it seemed to work fine (it completes `(` with `)`, but if you then insert a `*`, it matches `(*` and inserts only the missing `*`.

---

Nice! I've committed your suggestion :smile: